src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -157,7 +180,30 @@ def _create_temporary_field(
     return Field(field_node, field_type)
 
 
-def translate_as_field_op(
+def translate_domain(node: gtir.Node) -> FieldopDomain:
```

Comment:
This is complete optional, but I do not like `translate` how about `process_domain()` or `extract_domain()`?